composer.json

```
@@ -76,13 +76,15 @@
         "liip/imagine-bundle": "^1.9 || ^2.0",
         "matthiasnoback/symfony-dependency-injection-test": "^3.1",
         "nelmio/api-doc-bundle": "^2.11",
+        "sensio/framework-extra-bundle": "^3.0.13 || ^4.0 || ^5.0",
```

```suggestion
        "sensio/framework-extra-bundle": "^5.0",
```
We can just require the last one as this is new